build.fsx

```diff
@@ -132,9 +137,59 @@ Target "RunTests" <| fun _ ->
     let xunitToolPath = findToolInSubPath "xunit.console.exe" "src/packages/xunit.runner.console*/tools"
     printfn "Using XUnit runner: %s" xunitToolPath
     xUnit2
-        (fun p -> { p with OutputDir = testOutput; ToolPath = xunitToolPath })
+        (fun p -> { p with HtmlOutputPath = Some(testOutput @@ "xunit.html") })
         xunitTestAssemblies
 
+Target "StartDbContainer" <| fun _ ->
+    let dockerImage = getBuildParamOrDefault "dockerImage" @"microsoft/mssql-server-windows-express"
+    logfn "Starting SQL Express Docker container using image: %s" dockerImage
+    let posh = PowerShell.Create().AddScript(sprintf @"./docker_sql_express.ps1 -dockerImage %s" dockerImage)
+    posh.Invoke() |> Seq.iter (logfn "%O")
+
+    match posh.HadErrors with
+    | true -> posh.Streams.Error |> Seq.iter (logfn "\t %O")
+              failwith "SQL Express Docker container startup encountered an error... failing build"
+    | false -> ()
+
+    match environVarOrNone "container_ip" with
+    | Some x -> logfn "SQL Express Docker container created with IP address: %s" x
+    | None -> failwith "SQL Express Docker container env:container_ip not set... failing build"
+
+Target "PrepAppConfig" <| fun _ -> 
+    let ip = environVarOrNone "container_ip"
+    match ip with
+    | Some ip ->
+        let appConfig = "src/Akka.Persistence.SqlServer.Tests/App.config"
+        let configFile = readConfig appConfig
+        let connStringNode = configFile.SelectSingleNode "//connectionStrings/add[@name='TestDb']"
+        let connString = connStringNode.Attributes.["connectionString"].Value
+
+        log ("Existing App.config connString: " + Environment.NewLine + "\t" + connString)
+
+        let newConnString = new DbConnectionStringBuilder();
+        newConnString.ConnectionString <- connString
+        newConnString.Item("data source") <- ip
+    
+        log ("New App.config connString: " + Environment.NewLine + "\t" + newConnString.ToString())
+
+        updateConnectionString "TestDb" (newConnString.ToString()) appConfig
+        CopyFile "src/Akka.Persistence.SqlServer.Tests/bin/Release/Akka.Persistence.SqlServer.Tests.dll.config" appConfig
+
+    | None -> failwith "SQL Express Docker container not started successfully $env:container_ip not found... failing build"
+
+FinalTarget "TearDownDbContainer" <| fun _ ->
+    match environVarOrNone "container_name" with
```

Comment:
Checking the build logs, was able to verify that the container does actually shut down correctly:

```
[03:49:11]	[Step 1/1] Starting FinalTarget: teardowndbcontainer
[03:49:14]	[Step 1/1] Killing container: akka-78869d2b-5c52-4951-bcdd-d7955a3e15c4
```